superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -852,25 +852,36 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     setTabKey(key);
   };
 
-  const renderStepTwoAlert = () =>
-    db?.engine && (
-      <StyledAlertMargin>
-        <Alert
-          closable={false}
-          css={(theme: SupersetTheme) => antDAlertStyles(theme)}
-          type="info"
-          showIcon
-          message={
-            engineSpecificAlertMapping[db.engine]?.message ||
-            connectionAlert?.DEFAULT?.message
-          }
-          description={
-            engineSpecificAlertMapping[db.engine]?.description ||
-            connectionAlert?.DEFAULT?.description
-          }
-        />
-      </StyledAlertMargin>
+  const renderStepTwoAlert = () => {
+    const path = window.location.hostname;
+    let ipAlert = connectionAlert.REGIONAL_IPS.default;
+    const regionalIPs = Object.keys(connectionAlert.REGIONAL_IPS);
+    regionalIPs.forEach(regex => {
+      if (regex.match(path)) {
+        ipAlert = connectionAlert.REGIONAL_IPS[regex];
+      }
+    });
```

Comment:
You can iterate over key:vale pairs directly:

```suggestion
    const regionalIPs = connectionAlert?.REGIONAL_IPS || {};
    Object.entries(regionalIPs).forEach(([regex, ipRange]) => {
      if (regex.match(hostname)) {
        ipAlert = ipRange;
      }
    });
```